openedx/features/marketplace/tests/view_tests.py

```diff
@@ -19,14 +21,38 @@ def setUpClass(cls):
         theme = SiteTheme(site=site, theme_dir_name='philu')
         theme.save()
 
+    def setUp(self):
+        super(MarketplaceViewTests, self).setUp()
+        self.user = UserFactory()
```

Comment:
Let's please use `user = UserFactory()` in `test_marketplace_request_form` as it isn't something that is being used in multiple test cases.

Reply:
@zeshanarifios we are using it in `test_get_marketplace_make_request_page` as well will also using in other tests as development progress.